### PR TITLE
net-p2p/deluge: Fix deluge-web systemd service unit

### PR DESCRIPTION
**Pull request by PPed72**
Closes: https://bugs.gentoo.org/888183
Signed-off-by: Paolo Pedroni <paolo.pedroni@iol.it>

### CODE REVIEW

File: net-p2p/deluge/deluge-2.1.1-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+	xdg_pkg_postinst
+
+	elog
+	elog "If, after upgrading, deluge doesn't work please back up and then"
+	elog "remove your '~/.config/deluge' directory and try again"
+	elog
+	elog "To start the daemon either run 'deluged' as user"
+	elog "or modify /etc/conf.d/deluged and run"
+	elog "/etc/init.d/deluged start as root"
+	elog "You can still use deluge the old way"
```
Comment (kuraga) [inline net-p2p/deluge/deluge-2.1.1-r2.ebuild:162]: @PPed72 , should we add something about systemd services?

Comment (PPed72) [reply net-p2p/deluge/deluge-2.1.1-r2.ebuild:162]: That might be a nice idea ;)
On it...

**Comment by PPed72** [comment]
@kuraga , better?

**Comment by kuraga** [comment]
@PPed72 , excellent!

**Comment by PPed72** [comment]
What about merging, then?

**Review by juippis — APPROVED** [review]

**Comment by kuraga** [comment]
@juippis , @PPed72 , it's very bad that we have removed the only stable version...

**Comment by juippis** [comment]
Oh, I did **not** see that. I'll fix, thanks!

**Comment by kuraga** [comment]
> Oh, I did **not** see that. I'll fix, thanks!

Me too :(

**Comment by PPed72** [comment]
-r1 is stable as well, isn't it?

**Comment by juippis** [comment]
Yeah everything looks fine in my tree.

**Comment by kuraga** [comment]
Yes, sorry. Renaming confused me. Sorry.

**Comment by PPed72** [comment]
BTW, the renaming is a side effect of squashing a few different commits for the pull request.

**Pull request closed without merging**